lib/rules/require-proper-argument-of-trackCall.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+module.exports = {
+  meta: {
+    docs: {
+      description: 'All amplitude events should have proper capitalization',
+      category: 'Stylistic Issues',
+      recommended: false,
+    },
+    fixable: null, // or "code" or "whitespace"
+    schema: [],
+  },
+
+  create: function (context) {
+    return {
+      ExpressionStatement(node) {
+        if (context.getFilename().includes('.test.')) {
+          return undefined;
+        }
+        if (
+          !node ||
+          !node.expression ||
+          !node.expression.callee ||
+          !node.expression.arguments ||
+          !node.expression.arguments[0] ||
+          !node.expression.arguments[0].value
+        ) {
+          return undefined;
+        }
+        if (node.expression.callee.name !== 'trackCall') {
```

Comment:
I think moving this one up before checking for arguments would be faster as currently well be checking all function calls

Reply:
Higher we are checking if this exists, so I dont think we should be moving it higher in this case